rust/protocol/src/curve.rs

```diff
@@ -256,6 +269,7 @@ impl PrivateKey {
         }
     }
 
+    /// Calculate a new key agreed between this private key and the public key `their_key`.
```

Comment:
The result of this operation is usually known as a "shared secret" rather than a "key".

Reply:
Fixed!